site/src/pages/DeploymentSettingsPage/AppearanceSettingsPage/AppearanceSettingsPageView.tsx

```diff
@@ -112,7 +112,7 @@ export const AppearanceSettingsPageView: FC<
           corner of the dashboard."
 				validation={
 					isEntitled
-						? "We recommend a transparent image with 3:1 aspect ratio."
+						? "We recommend a transparent image with maximum 3:1 aspect ratio."
```

Comment:
this isn't very well worded imo. I'd change it to something like

> We recommend a transparent image. 
```suggestion
						? "An image with transparency and an aspect ratio of 3:1 or less will look best."
```